src/main/java/seedu/address/ui/FlashcardListPanel.java

```diff
@@ -32,10 +33,11 @@ public FlashcardListPanel(ObservableList<Flashcard> flashcardList) {
     /**
      * Creates a {@code FlashcardListPanel} with the given {@code ObservableList}.
      */
-    public FlashcardListPanel(ObservableList<Flashcard> flashcardList, boolean isQuiz) {
+    public FlashcardListPanel(ObservableList<Flashcard> flashcardList, boolean isNotAns, int currIndex) {
```

Comment:
Can we turn `isNotAns` to `isAns` and flip it inside the method? A bit counter intuitive to read haha

Reply:
I changed it to isQuestion, hopefully this is better!